dom.bs

```diff
@@ -2177,6 +2177,10 @@ steps:</p>
  <li><p>If the <i>open flag</i> is set and <var>shadow</var>'s <a for=ShadowRoot>mode</a> is
  <em>not</em> "<code>open</code>", then return null.</p></li>
 
+ <li><p>If <var>shadow</var>'s <a for=ShadowRoot>slot assignment</a> is "<code>manual</code>",
+ then return the <a>slot</a> in <var>shadow</var>'s <a for=tree>descendants</a> whose <a>manually assigned nodes</a>
+ <a for=set>contains</a> <var>slottable</var>, if any, and null otherwise.</p></li>
```

Comment:
The for=set here is questionable per https://github.com/whatwg/html/pull/5483#issuecomment-784464588